### PR TITLE
New tutorials for running deb or ubunt as user vms

### PR DESCRIPTION
**Pull request by deb-intel**
Signed-off-by: Deb Taylor <deb.taylor@intel.com>

### CODE REVIEW

File: doc/tutorials/running_deb_as_user_vm.rst [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+   .. code-block:: none
+
+      $ sudo mount /dev/sda1 /mnt
+      $ sudo sed -i "s/0x01010F/0x010101/" /mnt/loader/entries/acrn.conf
+      $ sudo sed -i "s/0x011111110000/0x011100001111/" /mnt/loader/entries/acrn.conf
+      $ sed -i 3"s/$/ i915.enable_conformance_check=0/" /mnt/loader/entries/acrn.conf
+      $ sudo sync && sudo umount /mnt && reboot
+
+#. Copy grubx64.efi to bootx64.efi:
```
Comment (gvancuts) [inline doc/tutorials/running_deb_as_user_vm.rst]: @lirui34 , I forgot to ask earlier but why is this operation needed (copying `grubx64.efi` to `bootx64.efi`). Is OVMF.fd not able to use Grub directly?

Comment (lirui34) [reply doc/tutorials/running_deb_as_user_vm.rst]: Yes, Debian VM not able to launch with the OVMF.fd unless we rename the `grubx64.efi` to `bootx64.efi`.

Comment (gvancuts) [reply doc/tutorials/running_deb_as_user_vm.rst]: Interesting, do you know why that's only happening with Debian but not with Ubuntu?

Comment (lirui34) [reply doc/tutorials/running_deb_as_user_vm.rst]: I captured original contents of `ubuntu.img` and `debian.img`, for the debian vm, it's missing boot folder by default:
```
/mnt/EFI/
└── debian
    ├── BOOTX64.CSV
    ├── fbx64.efi
    ├── grub.cfg
    ├── grubx64.efi
    ├── mmx64.efi
    └── shimx64.efi

/mnt/EFI/
├── BOOT
│   ├── BOOTX64.EFI
│   └── fbx64.efi
└── ubuntu
    ├── BOOTX64.CSV
    ├── fw
    ├── fwupx64.efi
    ├── grub.cfg
    ├── grubx64.efi
    ├── mmx64.efi
    └── shimx64.efi
```

**Pull request merged**